runtime/mainnet/src/governance/tracks.rs

```diff
@@ -4,6 +4,9 @@ const fn percent(x: i32) -> sp_arithmetic::FixedI64 {
 	sp_arithmetic::FixedI64::from_rational(x as u128, 100)
 }
 use pallet_referenda::Curve;
+
+use runtime_common::prod_or_fast;
```

Comment:
prod is for mainnet only I guess, correct?

Reply:
Yeah.